### ISSUE TITLE
Future news visible in Galaxy webhook

### ISSUE THREAD

**Issue by bebatut** [usegalaxy-eu/website#732]
Seen today (05.07.21) in Galaxy webhook. See the first 2 news:

![image](https://user-images.githubusercontent.com/1842467/124428592-324ccd80-dd6d-11eb-9e2a-58562c023caa.png)

**Comment by beatrizserrano** [comment]
yes, talked about it with @bgruening recently but couldn't reproduce it, thanks!

**Comment by OlegZharkov** [comment]
what is the problem here? Are we supposed to hide them?
